extensions/oper-tag.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "`oper-tag` Extension"
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "David Schultz"
+    period: "2022"
+    email: "me@zpld.me"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `oper` tag name. Instead, implementations SHOULD use the
+`draft/oper` tag name to be interoperable with other software
+implementing a compatible work-in-progress version.
+
+## Description
+
+The `draft/oper` capability causes the server to add a [message tag][] to messages sent by a user who is currently an IRC operator.
+
+The tag MUST be named `draft/oper`. The value of the tag, if specified, MUST identify the operator.
+
+The tag MUST be added by the ircd to all commands sent by a user (e.g. PRIVMSG,
+MODE, NOTICE, and all others) and numeric replies sent on behalf of the user.
```

Comment:
Ref https://github.com/ircv3/ircv3-specifications/pull/505: maybe turn this into a SHOULD for other numerics.